airflow/providers/amazon/aws/hooks/sqs.py

```diff
@@ -22,7 +22,7 @@
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
 
 
-class SQSHook(AwsBaseHook):
+class SqsHook(AwsBaseHook):
```

Comment:
This is also not backward compatible. We want both `SQSHook` and `SqsHook` to work.

Reply:
This has been handled please check and let me know what you think.